### ISSUE TITLE
Animation of semitransparent PNG images in Internet Explorer

### ISSUE THREAD

**Issue by alexeych** [malsup/cycle#31]
When cycle plugin effects are used with semitransparent PNG images, Internet Explorer (all versions, except IE9) makes semitransparent pixels opaque. 
The solutions is to add void "filter" property for IE during animation.
I used 'custom' effect with animIn and animOut options with properties {filter: '', ...} 
But it would be great to add this hack (only for IE) to all effects by default.

**Comment by malsup** [comment]
But not everyone is using pngs with alpha.

**Comment by alexeych** [comment]
Whose who don't use PNG with alpha will not be affected anyway. But who does (as such PNGs are more and more common in web-design), will not be stuck with this problem.
Or may be add this hack as a new option.

**Comment by YDiezijn** [comment]
I agree with alexeych, you 've written such a good plugin, but semi transparent pngs are here to stay.
I tried everything I could find on the web(your site,stackoverflow,jquery forum to name a few) to have the slideshow working with semi transparent pngs, but nothing worked.
Maybe" not everyone is using pngs with alpha", but for the ones who,please reconsider.
